### DISCUSSION TITLE
Limited color choices in 2.17.0

### DISCUSSION THREAD

**Discussion by npostnik** [kimai/kimai#4887] [Questions & Answers]
Hello,
according to the documentation here https://www.kimai.org/documentation/configurations.html#theme it should be possible to disable the limited color choices. But I cannot find the option to unlimit the color choices anywhere. Am I missing something?

**Comment by kevinpapst** [comment]
Thanks for bringing this up. This config was removed with 2.0.
I updated the docs to reflect reality.
